sdks/java/io/hadoop-format/build.gradle

```diff
@@ -47,6 +47,8 @@ configurations.testRuntimeClasspath {
 
   // Force use the old version of JAMM that cassandra relies on
   resolutionStrategy.force 'com.github.jbellis:jamm:0.3.0'
+  // Pin snakeyaml version due to cassandra-all does not support 2.x
+  resolutionStrategy.force 'org.yaml:snakeyaml:1.33'
```

Comment:
sorry, just realize this could affect both main and test artifact. Is it possible to just pin the dependency for test (so it won't saffect the released artifacts to be released)

Reply:
ah this is testRuntimeClasspath. never mind, LGTM